src/UserService.Business/Commands/User/EditUserCommand.cs

```diff
@@ -41,7 +43,7 @@ public class EditUserCommand : IEditUserCommand
 
     #region private method
 
-    private void EditCompanyEmployee(bool removeDepartment, Guid? departmentId, Guid? positionId, Guid? officeId, Guid userId, List<string> errors)
+    private async Task EditCompanyEmployee(bool removeDepartment, Guid? departmentId, Guid? positionId, Guid? officeId, Guid userId, List<string> errors)
     {
       string departmentErrorMessage = $"Cannot assign position to user. Please try again later.";
```

Comment:
fix type please, department not position in message

Reply:
value first and second variable are switched

Reply:
++